### DISCUSSION TITLE
Xcom : WoTC on Linux using AML

### DISCUSSION THREAD

**Discussion by TRNEEDANAME** [X2CommunityCore/xcom2-launcher#296] [Help]
**AML Version:** 
1.4.1, new install, never installed AML before on this machine

**Description**
I have launched it with Wine, I got this after hitting the "play" button : 
AML version: 1.4.1
Sentry GUID: 7922776e-e08a-4251-a9cd-9397ec0d76a6
Message: Could not load file or assembly 'Steamworks.NET, Version=13.0.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies.

Stack: 
   at XCOM2Launcher.Program.Main () [0x000b9] in <4e4cf5d57d0b4e6480dec879c6a9aefd>:0

**Comment by RevZero** [comment]
Linux is not officially supported, so I will convert this issue to a discussion.

As the error indicates, it seems that the `Steamworks.NET.dll` could not be loaded. Make sure the file exists and that you got the correct version. Maybe you updated from an earlier AML and did not overwrite the old file.
